src/Core/View.js

```diff
@@ -439,7 +477,12 @@ View.prototype.addFrameRequester = function addFrameRequester(when, frameRequest
  * @param {FrameRequester} frameRequester
  */
 View.prototype.removeFrameRequester = function removeFrameRequester(when, frameRequester) {
-    this._frameRequesters[when].splice(this._frameRequesters[when].indexOf(frameRequester), 1);
+    const index = this._frameRequesters[when].indexOf(frameRequester);
```

Comment:
it's a bug fix or a enhancement? Is it necessary for PR?
just to understand

Reply:
It's a bug fix, that'll go in its own commit as soon as the rest of the PR is ok.